api/domain/index.test.js

```diff
@@ -0,0 +1,176 @@
+import express from "../../express";
+import routes from "..";
+import { createUserAndDomain, dropTables } from "../../test/db-prepare";
+import Domain from "../../models/domain";
+import request from "supertest";
+import mongoose from "mongoose";
+
+const app = () => express(routes);
+
+let cred;
+beforeEach(async done => {
+  await dropTables();
+  cred = await createUserAndDomain(app);
+  done();
+});
+
+describe("Contact", () => {
+  it("should create a new domain with default settings", async () => {
+    const domain = await Domain.findById(cred.domainId);
+
+    const expectedSettings = {
+      customFields: [
+        {
+          model: "Contact",
+          name: "Phone",
+          type: "string",
+          isAlwaysVisible: true,
+          isShownInAddDialog: false,
+          isDefault: true,
+        },
+        {
+          model: "Contact",
+          name: "Email",
+          type: "string",
+          isAlwaysVisible: true,
+          isShownInAddDialog: false,
+          isDefault: true,
+        },
+        {
+          model: "Organization",
+          name: "Address",
+          type: "string",
+          isAlwaysVisible: true,
+          isShownInAddDialog: false,
+          isDefault: true,
+        },
+      ],
+    };
+
+    expect(domain.settings.customFields).toHaveLength(expectedSettings.customFields.length);
```

Comment:
`expectedSettings` is  using only for length, use const number instead or check on equals or smth

Reply:
Agree with @SamardakOleh. We should check all default fields. Otherwise there is no reason to construct complete  array to check only its length.